base_product_mass_addition/tests/test_product_mass_addition.py

```diff
@@ -36,6 +36,11 @@ def setUpClass(cls):
             **cls.quick_ctx
         )
 
+    @classmethod
+    def tearDownClass(cls):
+        cls.loader.restore_registry()
+        return super().tearDownClass()
```

Comment:
It's better to register the call `restore_registry` method in the list of cleanup methods to execute just after the fake the registry is backed up
```
cls.loader.backup_registry()
cls.addClassCleanup(cls.loader.restore_registry)
``` 

In this way, even if an exception occurs into your setup, the registry will be cleaned up

Reply:
@lmignon Isn't it supported by test class ?

If not, odoo-test-helper documentation should be adapted too.

Reply:
@lmignon I have changed it